simplivity/resources/policies.py

```diff
@@ -89,6 +92,28 @@ def create(self, name, flags=None, timeout=-1):
         affected_object = self._client.do_post(URL, data, timeout, flags=flags)[0]
         return self.get_by_id(affected_object["object_id"])
 
+    def resume(self, target=None, timeout=-1):
+        """Resumes policy-based backups on a specific targeted object
+
+            Args:
+                target: Target object.
+                        Allowed object of host, omnistack_cluster, cluster_group and default is federation
+                timeout: Time out for the request in seconds.
+
+            Returns:
+                None
+        """
+
+        data = {}
+        if isinstance(target, (Host, OmnistackCluster, ClusterGroup)):
+            data["target_object_type"] = target.OBJECT_TYPE
```

Comment:
I don't think we need an extra class variable to find the type of an object.

Reply:
@sijeesh  , We tried using type function to get the object type however value is simplivity.resources.omnistack_clusters.OmnistackCluster, whereas we need the value as 'omnistack_cluster' so we cannot really use type function also isinstance gives us boolean output. In summary we were not able to use the programmatic functions to retrieve the value and hence we declared OBJECT_TYPE variable.

Reply:
Commented on other PR. as we are planning to use this class variable to fetch the data from the response body, lets go with this approach.

Reply:
Let update Policy class also to add OBJECT_TYPE and use it in the refresh method.

Reply:
Added OBJECT_TYPE variable and used it in the refresh method.